pkg/services/ngalert/state/historian/annotation.go

```diff
@@ -179,8 +196,13 @@ func (h *AnnotationBackend) recordAnnotationsSync(ctx context.Context, panel *pa
 		}
 	}
 
+	org := fmt.Sprint(orgID)
+	h.metrics.WritesTotal.WithLabelValues(org).Inc()
+	h.metrics.TransitionsTotal.WithLabelValues(org).Add(float64(len(annotations)))
```

Comment:
I just realised you've chosen to not attempt to measure bytes when you're using the old annotation backend - could you clarify this in the help of the metric?

Reply:
Good catch, I did the same for the request histogram which also only applies to external stores.